CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to this repository
+
+# Table of Contents
+1. [Commits and pull requests](#commits-release-notes-pull-requests)
+    1. [Fixing a bug](#fixing-a-bug)
+    2. [Commits](#commits)
+    3. [Pull requests](#pull-requests)
+        1. [Reviewing pull requests](#reviewing-pull-requests)
+        2. [Merging pull requests](#merging-pull-requests)
+
+## Commits and pull requests<a name="commits-release-notes-pull-requests"></a>
```

Comment:
I don't think these anchors are needed, GFM anchors the headings by default, so you should be able to link to another section using `#` followed by the heading lowercased with spaces replaced by dashes eg `[link text](#commits-and-pull-requests)`. 

Also html5 depreciates the name attribute in favour of id.